### PR TITLE
Fix setOVADefault to set size of OVA in ovf file

### PR DESCRIPTION
**Pull request by Rebits**
Hello team,

I have changed setOVADefault.sh script. Now it set size properly. Also, I have fixed a typo error in kibana package URL.

### CODE REVIEW

File: ova/wazuh_ovf_template [hunk 1]
```diff
@@ -1,109 +1,110 @@
...
+        <rasd:AutomaticAllocation>false</rasd:AutomaticAllocation>
+        <rasd:Description>Floppy Drive</rasd:Description>
+        <rasd:ElementName>Floppy 1</rasd:ElementName>
+        <rasd:InstanceID>8</rasd:InstanceID>
+        <rasd:ResourceType>14</rasd:ResourceType>
+      </Item>
+    </VirtualHardwareSection>
+  </VirtualSystem>
+</Envelope>
+
```
Comment (DFolchA) [inline ova/wazuh_ovf_template:110]: Remove empty last line

File: ova/setOVADefault.sh [hunk 2]
```diff
@@ -38,3 +38,4 @@ echo "SHA1(${file}.ovf) = ${sha_ovf}" >> ${workspace}/new-ova/${file}.mf
 
 tar -cvf "${dest_ova}" -C "${workspace}/new-ova/" ${file}.ovf ${file}-disk-1.vmdk ${file}.mf
 rm -rf ${workspace}/new-ova/
+
```
Comment (DFolchA) [inline ova/setOVADefault.sh:41]: Remove the last line

**Review by DFolchA — CHANGES_REQUESTED** [review]

File: ova/wazuh_ovf_template (hunk 1 above)
Comment (Rebits) [reply ova/wazuh_ovf_template]: Done in e1ed9fda43e7bc66efb1c01f56d1564c867acdee

File: ova/setOVADefault.sh (hunk 2 above)
Comment (Rebits) [reply ova/setOVADefault.sh]: Done in e1ed9fda43e7bc66efb1c01f56d1564c867acdee

**Review by BraulioV — APPROVED** [review]

**Review by DFolchA — APPROVED** [review]
LGTM

**Review by BraulioV — APPROVED** [review]

**Pull request merged**